app/build.gradle.kts

```diff
@@ -33,6 +37,12 @@ android {
                 getDefaultProguardFile("proguard-android-optimize.txt"),
                 "proguard-rules.pro"
             )
+            buildConfigField("String", "BASE_URL", "\"https://tasky.pl-coding.com/\"")
```

Comment:
You are defining the url here and on debug, but its already defined on "defaultConfig"

Theres no need to add it on debug and release


Same with the apikey. You can define it on defaultConfig, and remove from debug and release

Reply:
Yes, I made a mistake here